src/config.py

```diff
@@ -1,16 +1,33 @@
 import os
+import re
 import yaml
 from typing import Optional
 import streamlit as st
 
-from pydantic import BaseModel, model_validator
+from pydantic import BaseModel, field_validator
 
 class EvaluationConfig(BaseModel):
     name: str
     default_scorer: str
     default_metric: str
     paths: list[str]
 
+    @field_validator('paths')
+    @classmethod
+    def substitute_env_vars(cls, paths: list[str]) -> list[str]:
```

Comment:
Could you write a test for this?

Reply:
Or if you're writing tests in a separate PR in a batch that's fine as well.

Reply:
I'll write it here!

Reply:
Let's move tests to the root (same level as src)